styx-scheduler-service/src/main/java/com/spotify/styx/ServiceAccountKeyManager.java

```diff
@@ -101,7 +101,7 @@ public void deleteKey(String keyName) throws IOException {
     } catch (GoogleJsonResponseException e) {
       // TODO: handle 403 correctly once google fixes their API
       if (e.getStatusCode() == 403 || e.getStatusCode() == 404) {
-        LOG.debug("Couldn't find key to delete {}", keyName);
+        LOG.debug("Couldn't find key to delete {}", keyName, e);
```

Comment:
Don't think the stack trace is needed here, it's not really an exception.

Reply:
On other 404 checks I did consider the cause captured without additional logging, but because here there's even a TODO about the 403 situation I went the pedantic route. One bit of information captured :]